src/domUpdates.js

```diff
@@ -160,16 +161,62 @@ const pageLoadRenders = () => {
   renderTagArea();
 };
 
-const showRecipe = (recipeCard) => {
+const getInstructionHTML = (recipe) => {
+  return recipe.instructions.map((instruction, i) => {
+    if(recipe.instructions.length > 1) {
+      return `<section class='single-instruction-step'> 
+                <p class='step'>STEP ${i+1}</p> 
+                <p class='instruction'>${instruction}</p> 
+              </section>`
+    } else {
+      return `<p>${instruction}</p>`
+    }
+  })
+}
+
+const addScrollBar = (recipe, element) => {
+  if (recipe.instructions.length > 1) {
+    document.querySelector(element).classList.add('scrollbar')
+  }
+}
+
```

Comment:
I just tested this: since we changed the scrollbar `overflow-y` to `auto`, I don't think we need to check if there is only 1 step anymore

Reply:
oh yeah! I will remove that if statement and leave it as a function so that we can add a scroll bar to the ingredients panel easily

Reply:
Ok done now!